elements/pfe-number/pfe-number.js

```diff
@@ -4,23 +4,23 @@ var commonjsGlobal =
   typeof window !== "undefined"
     ? window
     : typeof global !== "undefined"
-      ? global
-      : typeof self !== "undefined"
-        ? self
-        : {};
+    ? global
```

Comment:
@mwcz Do you want to pull this out of this PR since it's just prettifying a file that was not edited as part of this project?

Reply:
Sure.